test/vre/chapter31/config/militaryhistory.unit.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/vre/chapter31/config/form.js';
+
+describe('VRE chapter 31 military history', () => {
+  const { schema, uiSchema } = formConfig.chapters.militaryHistory.pages.militaryHistory;
+  it('renders military info', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={{}}
+        formData={{}}
+        uiSchema={uiSchema}/>
+    );
+
+    expect(form.find('input').length).to.equal(12);
+    expect(form.find('select').length).to.equal(5);
+  });
+
+  it('submits without info', () => {
+    const onSubmit = sinon.spy();
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={{
+          'view:isVeteran': true
+        }}
+        formData={{}}
+        onSubmit={onSubmit}
+        uiSchema={uiSchema}/>
+    );
+
+    form.find('form').simulate('submit');
+    expect(form.find('.usa-input-error').length).to.equal(0);
+
+    expect(onSubmit.called).to.be.true;
+  });
```

Comment:
I've been trying to write tests that add an item when the page has an array field.

Reply:
Thanks for pointing that out, added.